### PR TITLE
Scala 140

### PR DESCRIPTION
**Pull request by yadavan88**
Sample code for Slick queries

### CODE REVIEW

**Comment by yadavan88** [comment]
@glmartin I have moved the slick code samples and tests to separate sub-project.

File: scala-slick/src/test/scala/com/baeldung/scala/slick/PlayerServiceSpec.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import org.scalatest.concurrent.PatienceConfiguration.Timeout
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpec}
+import slick.jdbc.H2Profile.api._
+
+import scala.concurrent.duration._
+
+/**
+ * Created by yadu on 05/08/20
+ */
```
Comment (glmartin) [inline scala-slick/src/test/scala/com/baeldung/scala/slick/PlayerServiceSpec.scala:15]: Let's remove this comment.

File: scala-slick/src/main/scala/com/baeldung/scala/slick/SlickTables.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.baeldung.scala.slick
+
+import java.time.LocalDate
+
+/**
+ * Created by yadu on 04/08/20
+ */
```
Comment (glmartin) [inline scala-slick/src/main/scala/com/baeldung/scala/slick/SlickTables.scala:7]: Let's remove this comment.

File: scala-slick/src/main/scala/com/baeldung/scala/slick/PlayerService.scala [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.baeldung.scala.slick
+
+/**
+ * Created by yadu on 05/08/20
+ */
```
Comment (glmartin) [inline scala-slick/src/main/scala/com/baeldung/scala/slick/PlayerService.scala:5]: Let's remove this comment.

File: scala-slick/src/main/scala/com/baeldung/scala/slick/Entities.scala [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.baeldung.scala.slick
+
+import java.time.LocalDate
+
+/**
+ * Created by yadu on 02/08/20
+ */
```
Comment (glmartin) [inline scala-slick/src/main/scala/com/baeldung/scala/slick/Entities.scala:7]: Let's remove this comment.

File: scala-slick/src/main/scala/com/baeldung/scala/slick/Connection.scala [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package com.baeldung.scala.slick
+
+import slick.jdbc.H2Profile.api._
+/**
+ * Created by yadu on 02/08/20
+ */
```
Comment (glmartin) [inline scala-slick/src/main/scala/com/baeldung/scala/slick/Connection.scala:6]: Let's remove this comment.

**Pull request merged**